### ISSUE TITLE
Suggestions for new sounds

### ISSUE THREAD

**Issue by MyNameIsTroll** [ashutoshgngwr/noice#139]
Hey, everybody, very cool little app.

Some suggestions for new sounds :)

- fountain or cascade
- footsteps in the snow
- icebreaker

**Comment by MyNameIsTroll** [comment]
Other ideas ^^ (I'm addicted)

- the cry of the wolves
- kindergarten or recreation sounds
- train tracks
- cat purr
- bubble wrap

**Comment by ashutoshgngwr** [comment]
Thanks for the ideas. I should let you know that I have to salvage all the sounds from royalty-free online sources. Finding these sounds will hard, even impossible for some them. :sweat_smile:

**Comment by MyNameIsTroll** [comment]
That's what I thought :)
I wouldn't go to the forest to register wolves... :D

**Comment by ashutoshgngwr** [comment]
I might. You'll never know.

**Comment by igalic** [comment]
I gave a sleep helper toy from my daughter to a neighbour with a younger daughter, so after an evening of taking two hours to get her to sleep, I'd like to suggest:

- heartbeat

**Comment by oliver** [comment]
Another suggestion: a bird sample with less frequent sounds. The current birds.mp3 has basically continuous bird calls; and I'd love to have a sample where birds only call every 1 or 2 seconds maybe.

(Btw. many thanks for releasing this app - it is wonderful!)

**Comment by etam** [comment]
Another one: driving an electric car on a highway.

Possible source material: https://www.youtube.com/watch?v=XQuo4-t0p_8 (I can't say for sure, but I guess the author might give you samples under free license https://www.cgpgrey.com/contact)

**Comment by ashutoshgngwr** [comment]
All suggestions under consideration. I don't have a final timeline. Currently super occupied with Google cast #150.

Working branch: [_feat/google-cast_](https://github.com/ashutoshgngwr/noice/tree/feat/google-cast)

PS, I could use some help with the testing. If you guys are able to build the app and have a cast-enabled device, the latest commit on that branch is able to cast the playback. It doesn't have a working UI though.

**Comment by ashutoshgngwr** [comment]
Deferring all sound related changes to 0.9.x in favor of considering to work on #62 which would require rework on all sounds.

**Comment by baditaflorin** [comment]
@ashutoshgngwr love what you are doing here.
Do you know about this project https://freesound.org/?

**Comment by ashutoshgngwr** [comment]
@baditaflorin yeah. Noise wouldn't have been possible without it. :)

**Comment by amadeous** [comment]
Discovering your app and like it so far! Other suggestions to help review class material for exam:
- Sound during an exam
- Sound in a library
Thank you for the app! :smiley:

**Comment by pawlowskatze** [comment]
Hi, one more sound suggestion please:

- Seagull screaching

It would make a perfect companion sound for ocean waves sound ;)

**Comment by pawlowskatze** [comment]
Regarding my previous comment (seagull screaching):

I could probably create the desired sound myself - from multiple samples at freesound.org and collate them into a file and submit it here... if someone tells me what container, format, sampling rate and recommended volume level to use.

**Comment by ashutoshgngwr** [comment]
@pawlowskatze working on [dual samples](https://github.com/ashutoshgngwr/noice/tree/feat/dual-samples). Submitting sound now would mean that I'd have to re-encode everything later when this feature is complete. I am holding onto the suggestions for now. Once the feature is complete, I'll try to add as many suggestions as I can. For now, I think the best thing you can do is submit links to raw sample sources. That'd make it easier for me when I'm adding it

**Comment by ashutoshgngwr** [comment]
I have added whatever suggestions made sense to me. If you still think that I've missed something or have more suggestions, feel free to open up a new issue. Please do not add suggestions to existing issues. It becomes really hard to track.
